app/src/main/res/values/strings.xml

```diff
@@ -12,6 +12,7 @@
     <string name="nav_item_all_survey">Conference survey</string>
     <string name="nav_content_description_drawer_open">Open navigation drawer</string>
     <string name="nav_content_description_drawer_close">Close navigation drawer</string>
+    <string name="nav_content_description_drawer_header">DroidKaigi 2018</string>
```

Comment:
Can you add japanese? Probably It will warning 🇯🇵

Reply:
Oops, I'll add.